2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,29 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    if (!(this.volume === 100)) {
```

Comment:
The inner brackets here aren't actually needed, but they do make it a little more readable I think, code excellent, all tests passing